app/views/mentors/list.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/main.njk" %}
+
+{% set primaryNavId = "mentors" %}
+
+{% set title = "Mentors" %}
+
+{% block beforeContent %}
+
+{% endblock %}
+
+{% block content %}
+
+  {% include "_includes/notification-banner.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% include "_includes/page-heading.njk" %}
+
+      {{ govukButton({
+        text: "Add mentor",
+        href: actions.new
+      }) }}
+
+      {% if mentors.length %}
+        {% include "_includes/mentors/list.njk" %}
+      {% else %}
+        <p class="govuk-body">There are no mentors for {{ organisation.name if organisation.name else "your organisation" }}.</p>
```

Comment:
<p class="govuk-body">There are no mentors to show for {{ organisation.name if organisation.name else "your organisation" }}.</p>